tests/mongo/test_mongo.py

```diff
@@ -582,6 +583,11 @@ def test_status_unreachable(mongo_connector, mocker):
     )
 
 
+def test_get_version(mongo_connector):
+    # "in" and not "==", because sometimes, we can have "8.0.0.-blueeyes" as version
```

Comment:
I think the version should be cleaned by the connectors, as we will want strict comparisons later on. A semver regex will probably do the job for most connectors